.github/workflows/build-pkgs.yml

```diff
@@ -0,0 +1,54 @@
+name: Build Packages
+on:
+  push:
+    tags: '[0-9]+.[0-9]+.[0-9]+-*'
+  schedule:
+    - cron: '41 22 * * SUN'
+
+jobs:
+  build-rpm:
+    runs-on: ubuntu-latest
+    env:
+      # _github_home is the dir used as a volume by the container
+      RPMDIR: /home/runner/work/_temp/_github_home/rpmbuild
+    steps:
+    - name: Install rpmlint
+      run: |
+        sudo apt-get update
+        sudo apt-get install rpmlint
+    - uses: actions/checkout@v2
+      with:
+        # Get all branches and tags so the latest tag can be found for VERSION
+        fetch-depth: 0
+    - name: Set VERSION
+      run: echo "VERSION=`echo $(git describe --tags $(git rev-list --tags --max-count=1))`" >> $GITHUB_ENV
```

Comment:
Can you get the tag that is being built from the [GitHub context](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context)? Specifically `github.ref`.

Reply:
No, as that would give `refs/tags/<tag_name>` rather than just `<tag_name>`.